lib/src/screens/receive/receive_page.dart

```diff
@@ -220,11 +221,16 @@ class ReceivePage extends BasePage {
         children: [
           Expanded(
             flex: 7,
-            child: QRWidget(
-                addressListViewModel: addressListViewModel,
-                isAmountFieldShow: true,
-                amountTextFieldFocusNode: _cryptoAmountFocus,
-                isLight: currentTheme.type == ThemeType.light),
+            child: Observer(
+              builder: (context) {
+                return QRWidget(
+                    addressListViewModel: addressListViewModel,
+                    isAmountFieldShow: true,
+                    urlString: addressListViewModel.uri.toString(),
+                    amountTextFieldFocusNode: _cryptoAmountFocus,
+                    isLight: currentTheme.type == ThemeType.light);
+              }
+            ),
```

Comment:
since we will revert changes in QR widget, no need for the Observer parent, and no need for urlString since it will be removed